bounties/level-1/basic-account-viewer.md

```diff
@@ -0,0 +1,44 @@
+# Basic account viewer
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 200 XLM | 1 Credits | Continuos
```

Comment:
Given the history of completed bounties I'd rather put this in the mid/upper L2 range.

Reply:
I see what you mean. I think it's fine though, the acceptance criteria for this bounty is simple so I still think it should be level 1.

Reply:
Let's leave this as L1 for now and see how it lands.